Comparing source compatibility of /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-http/2.12.4-SNAPSHOT/jspwiki-http-2.12.4-SNAPSHOT.jar against /home/jenkins/.m2/repository/org/apache/jspwiki/jspwiki-http/2.12.0/jspwiki-http-2.12.0.jar
WARNING: You are using the option '--ignore-missing-classes', i.e. superclasses and interfaces that could not be found on the classpath are ignored. Hence changes caused by these superclasses and interfaces are not reflected in the output.
===  UNCHANGED CLASS: PUBLIC org.apache.wiki.http.filter.CsrfProtectionFilter  (not serializable)
	===  CLASS FILE FORMAT VERSION: 55.0 <- 55.0
	===  UNCHANGED SUPERCLASS: java.lang.Object (<- java.lang.Object)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ANTICSRF_PARAM
	===  UNCHANGED CONSTRUCTOR: PUBLIC CsrfProtectionFilter()
	===  UNCHANGED METHOD: PUBLIC void destroy()
	===  UNCHANGED METHOD: PUBLIC void doFilter(javax.servlet.ServletRequest, javax.servlet.ServletResponse, javax.servlet.FilterChain)
		===  UNCHANGED EXCEPTION: java.io.IOException
		===  UNCHANGED EXCEPTION: javax.servlet.ServletException
	===  UNCHANGED METHOD: PUBLIC void init(javax.servlet.FilterConfig)
	===  UNCHANGED METHOD: PUBLIC STATIC boolean isCsrfProtectedPost(javax.servlet.http.HttpServletRequest)

Semantic versioning suggestion: 0.0.1